client/my-sites/site-settings/section-import.jsx

```diff
@@ -146,21 +149,28 @@ class SiteSettingsImport extends Component {
 							</header>
 						</CompactCard>
 
-						{ ( ! hasMediumActiveImports || hasWpActiveImports ) &&
+						{ ( ( ! hasMediumActiveImports && ! hasBloggerActiveImports ) || hasWpActiveImports ) &&
```

Comment:
hrm. these are looking a bit silly. should we be consolidating these into a single selector?

Reply:
Yes this was spotted as it was getting worked on, but the decision was made to break the changes up in order to for the PRs to each have a single purpose.

@talldan addressed this problem in #24349 which I'm looking to update and merge.